src/types/debate.ts

```diff
@@ -1,5 +1,12 @@
-const displayImageTypeArray = ["null", "MOW2024", "MOW2018", "custom"] as const;
 import { defaultSoundPack, soundPack } from "./soundPack";
+
+const displayImageTypeArray = [
+  "null",
+  "MOW2024",
+  "MOW2018",
+  "PND2024",
```

Comment:
Please rename this option, as described in [#68](https://github.com/debatecore/debate-tools/issues/68).